packages/dd-trace/src/plugins/util/web.js

```diff
@@ -98,7 +98,7 @@ const web = {
     const span = tracer.startSpan(name, { childOf })
 
     span.addTags({
-      [RESOURCE_NAME]: middleware._name || middleware.name || '<anonymous>'
+      [RESOURCE_NAME]: middleware._name || middleware.name || name
```

Comment:
This line should be setting a tag with the name `resource.name` to the name of the middleware, if it has a name. If it doesn't have a name (i.e.) it is an anonymous function then the name will be `<anonymous>` to indicate that. How does this impact the operation name?